backends/p4tools/testgen/targets/bmv2/table_stepper.cpp

```diff
@@ -154,6 +154,105 @@ void BMv2_V1ModelTableStepper::evalTableActionProfile(
     }
 }
 
+void BMv2_V1ModelTableStepper::evalTableActionSelector(
+    const std::vector<const IR::ActionListElement*>& tableActionList) {
+    const auto* keys = table->getKey();
+    // If we have no keys, there is nothing to match.
+    if (keys == nullptr) {
```

Comment:
Do we need to handle the default action, even in this case?

Reply:
Thanks for catching that, fixed. 

We have no support for default action override yet, which is why we return here. We should consolidate this code once we implement default action overrides.